Alloy/commands/compile/compilerUtils.js

```diff
@@ -212,6 +212,18 @@ exports.getParserArgs = function(node, state, opts) {
 			});
 		} else {
 			var theValue = node.getAttribute(attrName);
+
+			// find platform specific attributes
+			var attributeParts = attrName.split(':');
+			if ( attributeParts.length === 2 ) {
```

Comment:
Similar to line 164, we nee to check if the suspected platform `attributeParts[0]` is in `CONST.PLATFORMS`. Otherwise we are ignoring setting properties on `ListItems` combined with an `ItemTemplate`


```suggestion
			if ( attributeParts.length === 2 && _.includes(CONST.PLATFORMS, attributeParts[0])) {
```